provider/datagen/src/transform/cldr/datetime/mod.rs

```diff
@@ -103,6 +109,65 @@ macro_rules! impl_resource_provider {
                         data.eras.narrow.insert("2".to_string(), mundi_narrow.clone());
                     }
 
+                    if calendar == value!("japanese") {
```

Comment:
Question: The code here is showing as new, but it seems like this code must have already existed?

Reply:
No, previously we translated all Japanese eras unconditionally. We still do for `japanext`; `japanese` needs to filter out most of them.

The existing code is in Japanese's `get_era_code_map()` which is shared between the two calendars.